src/util.ts

```diff
@@ -282,7 +282,7 @@ export function deleteNestedProperty(obj: any, orderedProps: string[]) {
   if (orderedProps.length === 0) {
     return true;
   }
-  const prop = orderedProps.shift();
+  const prop = orderedProps.shift()!;
```

Comment:
This violates our linting rules. We should probably change the linting rules or add an exception here.

```suggestion
  const prop = orderedProps.shift()!; // eslint-disable-line @typescript-eslint/no-non-null-assertion
```

What do you think is better? 

Btw, I wish we could figure out a way to run the full test suite on forks but somehow it doesn't work yet. You can make branches in our repo as `krw/*` and send PRs from there.